pkgs/servers/nosql/ferretdb/default.nix

```diff
@@ -15,7 +15,8 @@ buildGoModule rec {
   };
 
   postPatch = ''
-    echo ${version} > internal/util/version/gen/version.txt
+    echo v${version} > build/version/version.txt
+    echo nixpkgs     > build/version/branch.txt
```

Comment:
There is no `nixpkgs` branch in the FerretDB repo, right? Am I interpreting the use of the term "branch" too literally?

From [version.go](https://github.com/FerretDB/FerretDB/blob/c3230915b4462373b6591ec56aa8b13eb04215a3/build/version/version.go#L19):

```go
// The following generated text files may be present in this (build/version) directory during building:
//   - version.txt (required) contains information about the FerretDB version in a format
//     similar to `git describe` output: `v<major>.<minor>.<patch>`.
//   - commit.txt (optional) contains information about the source git commit.
//   - branch.txt (optional) contains information about the source git branch.
```

Reply:
@noisersup Could you please address this?

Reply:
I updated that PR to use a new file that will be used by some future FerretDB release; see FerretDB/FerretDB#1806.